### PR TITLE
Implement params process data method

### PR DESCRIPTION
**Pull request by VkoHov**
- Implement params process data method
- Implement params getDataAsLines method

### CODE REVIEW

**Review by roubkar — APPROVED** [review]

**Comment by roubkar** [comment]
@VkoHov please check the brush functionality, it seems to be broken

**Comment by VkoHov** [comment]
> @VkoHov please check the brush functionality, it seems to be broken

File: aim/web/ui_v2/src/services/models/params/paramsAppModel.ts [hunk 1]
```diff
@@ -2,7 +2,27 @@ import paramsService from 'services/api/params/paramsService';
...
+} from 'types/services/models/metrics/metricsAppModel';
+import {
+  adjustable_reader,
+  decodePathsVals,
+  decode_buffer_pairs,
+  iterFoldTree,
+} from 'utils/encoder/streamEncoding';
+import { IRun } from 'types/services/models/metrics/runModel';
+import COLORS from 'config/colors/colors';
+import _ from 'lodash';
```
Comment (arsengit) [inline aim/web/ui_v2/src/services/models/params/paramsAppModel.ts:22]: please import lodash-es

**Review by KaroMourad — APPROVED** [review]
lgtm

**Review by arsengit — APPROVED** [review]

**Pull request merged**